package.json

```diff
@@ -137,7 +137,8 @@
     "watch": "node ./scripts/watch.js",
     "lingui:extract": "lingui extract",
     "lingui:compile": "lingui compile",
-    "postinstall": "yarn run typechain --target ethers-v5 --out-dir src/typechain src/abi/*.json src/abi/**/*.json"
+    "typechain:build": "yarn run typechain --target ethers-v5 --out-dir src/typechain src/abi/*.json src/abi/**/*.json",
+    "postinstall": "yarn typechain:build && yarn lingui:compile"
```

Comment:
are you ok with putting `yarn lingui:compile` in a postinstall hook? This would make it run every time a dev types `yarn` or `yarn install`

Reply:
Yeah I am okay. Lets talk about this with Jesusf since it has impacts on CI too. He added that:
`
    "prebuild": "git submodule update --init --recursive",
    "postbuild": "yarn lingui:compile",
`
Maybe it is better to have everything in postinstall